src/hook/useStringInput.tsx

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react';
+
+export const useStringInput = (initialStringValue: string = '') => {
```

Comment:
```suggestion
export const useStringInput = (initialStringValue = '') => {
```
타입 추론이 되기 때문에 이렇게 표현해도 될 것 같아요~

Reply:
> 타입 추론이 되기 때문에 이렇게 표현해도 될 것 같아요~

타입 추론이 될 부분도 고려해볼게요! 👍